### DISCUSSION TITLE
Navigator tab title duplicate using titleTemplate and NuxtContent

### DISCUSSION THREAD

**Discussion by Jocabin** [nuxt/nuxt#17112] [Questions]
Hello, I'm creating a Nuxt Content website, and I'm facing a problem.
The title template is duplicate in my navigator tab title, screenshot below :

![image](https://user-images.githubusercontent.com/48412786/183883131-a8ca50a0-d538-4bee-99dd-c8b78ee447ee.png)

In my `app.vue` file, I tried two methods :
```js
<script setup>
useHead({
  titleTemplate: (titleChunk) => {
    return `${titleChunk} - Title template`;
  },
});
</script>
```
or
```js
<script setup>
useHead({
  titleTemplate: "%s - Title template",
});
</script>
```

In my Markdown files, I define title like this : 
```md
---
title: "Title"
description: "Site desc"
---
```

I want to know if I define the title in my Markdown files the wrong way, or I need to report this as a bug.

If you need more informations, let me know.
Thanks in advance.

**Comment by brunomendesfx** [comment]
Hello,
here is also duplicating.

But the way I'm using it I find it more semantic to set the title on the pages:

in app.vue
```js
<script setup>
const route = useRoute()
const title = () => `My App - ${route.meta.title}`

useHead({
  title: title()
})

watchEffect(() => {
  useHead({
    title: title()
  })
})
</script>
```

pages:
```js
definePageMeta({
  title: 'Title page',
  layout: '....',
  middleware: '....',
})
```

so I can just use definePageMeta to define title, layout, middleware...

without needing to use useHead for title

**Reply by Jocabin** [reply]
Do you use this with Nuxt Content v2 ? Beacause the title template methods I show previously are actually working in Nuxt without Content.

**Reply by brunomendesfx** [reply]
use in the latest version of nuxt 3.

 as mentioned before I prefer to use this way than using the titleTemplate because I find it easier to then call the title, middleware, layout.... just in definePageMeta without having to use the separate useHead just for title

**Reply by Jocabin** [reply]
But how do you call `definePageMeta` in Markdown files ?

**Reply by brunomendesfx** [reply]
in app.vue
```vue
<template>
  <div>
    <NuxtPage />
  </div>
</template>

<script setup lang="ts">
const route = useRoute()
const title = () => `My App - ${route.meta.title}`

useHead({
  title: title()
})

watchEffect(() => {
  useHead({
    title: title()
  })
})
</script>
```

in pages/index.vue

```vue
<template>
  ...
</template>

<script setup lang="ts">
definePageMeta({
  title: 'Home'
})
</script>
```

in pages/login.vue

```vue
<template>
  ...
</template>

<script setup lang="ts">
definePageMeta({
  title: 'Login'
})
</script>
```

when you navigate between pages the title will change dynamically
